target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -293,11 +293,16 @@ netgear,r6120)
 	ucidef_set_led_wlan "wlan2g" "WiFi 2.4GHz" "$boardname:green:wlan2g" "phy0tpt"
 	;;
 netgear,r6220|\
-netgear,r6350|\
 netgear,wndr3700-v5)
 	ucidef_set_led_switch "wan" "wan" "$boardname:green:wan" "switch0" "0x10"
 	set_wifi_led "$boardname:green:wifi"
 	;;
+netgear,r6260|\
+netgear,r6350|\
+netgear,r6850)
+	ucidef_set_led_switch "wan" "wan" "$boardname:green:wan" "switch0" "0x10"
+	ucidef_set_led_wlan "wifi" "wifi" "$boardname:green:wifi" "phy0tpt"
```

Comment:
This userspace WiFi-LED definition can be dropped altogether, as the trigger is defined in the DTSI by default. The user can still change the LEDs behavior from userspace, however this redundancy can be dropped.

Reply:
yeah, but it will be missing in /etc/config/system and thus luci by default. i dont think thats very userfriendly. if you insist on dropping that line, i'll do so.

Reply:
This was topic in a ML thread some time ago as i've noted the same concern back then. As the board has the WiFi LED explicitl set, it's a property of the device (the user can change later by creating his/her own copnfiguration.

So yes, please drop it.

Reply:
Allright, didnt know that. Thanks for the explanation. Will keep that in mind for the next device :)